### ISSUE TITLE
Must click on first line to enter into text area in IE

### ISSUE THREAD

**Issue by Kryptonit3-zz** [OscarGodson/EpicEditor#316]
I noticed an issue when testing my implementation in IE, when I click anywhere besides the main "line" the box does not engage.

Here is a screenshot

![epiceditor](https://f.cloud.github.com/assets/5751754/2431081/9859e9aa-ad19-11e3-9d0f-306fcbde03e5.jpg)

**Comment by OscarGodson** [comment]
Hmm, yeah. I've noticed this too before. The easiest fix, I think, could be (on our side) whenever s person clicks into the editor always call focus() manually.

**Comment by jhofker** [comment]
Same for IE11. Looks like it also needs something like `cursor: text;` applied on the element so that the mouse isn't just an arrow.

**Comment by OscarGodson** [comment]
@jhofker oh, so even if you click in the middle you can begin typing? I thought it wasn't even focused?

**Comment by jhofker** [comment]
No, sorry @OscarGodson. :-) It works just like the screenshot and boxes that @Kryptonit3 included above. We have to click on that very first line in order to be able to edit or type anything.
